doc/examples/perLocationTransformSpreadsheet.gfr

```diff
@@ -0,0 +1,127 @@
+__children["Backdrop_Title"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -26.1708813 ), imath.V2f( 30, 10 ) ) )
+__children["Backdrop_Title"]["__uiPosition"].setValue( imath.V2f( -19.0039406, 1.4623704 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop_Tip"], 'nodeGadget:color', imath.Color3f( 0.300000012, 0.5, 0.460500002 ) )
+__children["Backdrop_Tip"]["title"].setValue( '' )
+__children["Backdrop_Tip"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -15.1441078 ), imath.V2f( 31.9457626, 19.6277027 ) ) )
+__children["Backdrop_Tip"]["__uiPosition"].setValue( imath.V2f( 30.6052666, -27.5567017 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop_Tip1"], 'nodeGadget:color', imath.Color3f( 0.300000012, 0.5, 0.460500002 ) )
+__children["Backdrop_Tip1"]["title"].setValue( 'Tip: Copying scene paths' )
+__children["Backdrop_Tip1"]["description"].setValue( "Once a spreadsheet is set up to drive a PathFilter node's paths plugs, you can save time by creating new rows directly based on location paths.\n\nTo copy scene paths as rows:\n\n1. Double-click the Spreadsheet node to open a Node Editor window focused on it.\n2. Select the node affected by the spreadsheet.\n3. In the Hierarchy View, select one or more locations.\n4. Click and drag the locations onto the add row button (+). For each location you dragged, a new row will be added, and will contain one of the paths." )
+__children["Backdrop_Tip1"]["__uiBound"].setValue( imath.Box2f( imath.V2f( -10, -15.1441078 ), imath.V2f( 20, 19.5885277 ) ) )
+__children["Backdrop_Tip1"]["__uiPosition"].setValue( imath.V2f( 74.1388779, -27.5825119 ) )
+Gaffer.Metadata.registerValue( __children["Backdrop_Tip2"], 'nodeGadget:color', imath.Color3f( 0.300000012, 0.5, 0.460500002 ) )
+__children["Backdrop_Tip2"]["title"].setValue( 'Tip: Driving PathFilter node paths' )
+__children["Backdrop_Tip2"]["description"].setValue( "To ensure the ShaderTweaks node only affects locations that match the spreadsheet rows, you must connect the paths plug of a PathFilter plug to the Spreadsheet:\n\n1. Select the PathFilter node.\n2. In the Node Editor, click the gear button, then select Create Spreadsheet or Connect to Spreadsheet > [spreadsheet name] from the context menu.\n\nThe spreadsheet's rows will now define the filter's paths, and the Selector plug will automatically be driven as `\\$\\{scene:path\\}`." )
```

Comment:
"To ensure the ~ShaderTweaks~ **Transform** node only" Sorry that was probably in my original suggestion.

Reply:
My mistake. 202ed056e12259507a6864b11bcfae6ad716bd16